lighthouse-core/audits/accessibility/aria-progressbar-name.js

```diff
@@ -19,7 +19,7 @@ const UIStrings = {
   /** Title of an accessibility audit that evaluates if progressbar HTML elements do not have accessible names. This title is descriptive of the failing state and is shown to users when there is a failure that needs to be addressed. */
   failureTitle: 'ARIA `progressbar` elements do not have accessible names.',
   /** Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'When an element doesn\'t have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-name/).',
+  description: 'When a `progressbar` element doesn\'t have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-name/).',
```

Comment:
> why this was previously passing

yep! this was the bug I was alluding to in https://github.com/GoogleChrome/lighthouse/pull/12697#discussion_r658384015 where the previous code only ever checked for `meaning` collisions with the *first* occurrence of a message and not between *all* occurrences of the message.

> not a TC issue

No clue. If I had to guess, it would be that they haven't needed to be translated yet at the *same time* so they weren't colliding in TC yet? But I have no clue why this has worked so far given the way TC has been explained to me.